lighthouse-core/lib/page-functions.js

```diff
@@ -148,22 +151,23 @@ function getOuterHTMLSnippet(element, ignoreAttrs = [], snippetCharacterLimit =
     // See https://github.com/GoogleChrome/lighthouse/issues/11465
     const template = element.ownerDocument.createElement('template');
     template.content.append(clone);
-    ignoreAttrs.concat(autoFillIgnoreAttrs).forEach(attribute =>{
+    autoFillIgnoreAttrs.forEach(attribute =>{
       clone.removeAttribute(attribute);
     });
     let charCount = 0;
     for (const attributeName of clone.getAttributeNames()) {
-      if (charCount > snippetCharacterLimit) {
+      let attributeValue = clone.getAttribute(attributeName);
+      if (attributeValue !== null) attributeValue = attrMapper(attributeName, attributeValue);
```

Comment:
Couldn't there be a situation where `attributeValue` is null but we want to override with an actual value?

Reply:
yea, I guess in that case the interface you'd want is `object -> object`. I don't think we consider this part of our public API so we can change this whenever we want.